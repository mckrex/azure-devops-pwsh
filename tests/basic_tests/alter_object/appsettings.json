{
    "object_1": {
        "enabled": false,
        "id": "7f1ae96f-1769-4d58-b636-ca0800c6550b",
        "maximum": 1,
        "minimum": 0.125
    }
}