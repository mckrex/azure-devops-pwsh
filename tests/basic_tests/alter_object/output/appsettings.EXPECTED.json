{
    "object_1": {
        "enabled": true,
        "id": "7f1ae96f-1769-4d58-b636-ca0800c6550b",
        "maximum": 1.667,
        "minimum": 0.667,
        "allowOffCycle": true
    },
    "object_2": {
        "enabled": false,
        "id": "c18359fb-30d0-4c6d-a73f-69d1ffca73e1",
        "maximum": 2,
        "minimum": 0.5
    }
}